assignment.js

```diff
@@ -66,6 +73,33 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+
+//Alternative using Math functions
+let numbers = [57, 13, 4, 38]
+currentLargestNumber = Math.max(...numbers)
+
+console.log("Using Math functions: ")
+console.log(currentLargestNumber);
+
+//Alternative with loops
+
+let numbers = [3, 48, 29, 12, 18]
+let currentLargestNumber = numbers[0]
+let arrIndex = 0
+
+for (let i = 1; i < numbers.length; i++) {
+  if (numbers[i] > currentLargestNumber) {
+    currentLargestNumber = numbers[i]
+    arrIndex = i
+  }
+}
+
+console.log("Using loops: ")
+console.log("The largest number is: " + `${currentLargestNumber}`)
+console.log("Located at array element number: " +  `${arrIndex}`)
+
```

Comment:
@AtleLohrmann Good Job here with your alternate approach. It's a good use of a for loop in combination with the conditional statements to evaluate what the largest value is.
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L73)